### ISSUE TITLE
Question about using Cifar100's dataset

### ISSUE THREAD

**Issue by ss3b3** [Sprinter1999/FedLSR#4]
I used your code for training on Cifar100 data and I also replicated your code myself for training on Cifar100. I find that both converge very slowly. I am guessing that there is something wrong with my hyperparameter settings, do you have any suggestions? Also if possible I would like to know your hyperparameter settings with Clothing1M.

**Comment by Sprinter1999** [comment]
Thank you ss3b3 for your issue. Sorry for this late reply, we don't implement CIFAR-100 experiments in the original FedLSR. There're some reasons for this:

- Knowledge distillation based methods often run into similar situation when the number of classes gets higher and I reckon, sharpen operation is not that useful at the same time. Maybe we can explore more methods and go beyond the hyper-param setting compared with searching for the best hyper-params.
- The role of $Loss_{reg}$ is also bounded if you check the underlying mathematical structure of $Loss_{reg}$. 
- Recently I'm also conducting many evaluation experiments on previous methods on CIFAR-100 , we can discuss more via emails.
- As for hyper-params of Clothing-1M, you can refer to the original paper. Maybe you'll see the acc on Clothing-1M firstly gets higher then goes down : ), you can implement co-teaching at the same time.

**Comment by ss3b3** [comment]
Thank you for your answer! I also think it might be that the sharpen operation is indeed less effective with many classes (I also found out in my experiments that the sharpen operation seems to have a negative effect ^_^).Looking forward to more of your excellent work in FL!

**Comment by Sprinter1999** [comment]
Thank you for your issue and support ^_^

Perhaps we can collaborate on Federated Learning against noisy labels together in the future~
